### PR TITLE
[NEW] Check if local branch is up to date with remote

### PR DESCRIPTION
**Pull request by elmatze79**

### CODE REVIEW

File: src/check_git_origin.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+####
+# Check the git origin if there are non merged/rebased commits
+####
+
+if [[ ! -d $1/.git ]]
+then
+    echo "Unknown: No valid git repository found in $1"
```
Comment (TheFRedFox) [inline src/check_git_origin.sh]: Consistent would be: 'UNKNOWN: ', uppercase

**Comment by TheFRedFox** [comment]
I am wondering for which reason this is used. I was looking at igmonplugins as some monitoring scripts for servers. Is it possible that there are local git repos on servers, which have to be checked then?

**Comment by elmatze79** [comment]
We have this one special project where I am not allowed to automatically have changes merged, so I need to know if there are changes to the repositories

File: src/check_git_origin.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+cd $1
+git fetch > /dev/null
+NUM_COMMITS=`git log --oneline HEAD..origin | wc -l`
+if [[ $NUM_COMMITS -eq 0 ]]
+then
+    echo "OK: $1 is up to date"
+    exit 0
+fi
+
+echo "Warning: $1 is $NUM_COMMITS commits behind origin"
```
Comment (TheFRedFox) [inline src/check_git_origin.sh]: Sry, but it is otherway around. Uppercase messages everywhere: UNKNOWN, OK, WARNING, CRITICAL

Comment (elmatze79) [inline src/check_git_origin.sh]: https://nagios-plugins.org/doc/guidelines.html#AEN78

Comment (TheFRedFox) [inline src/check_git_origin.sh]: Well, I checked all our scripts .. then we should do it in a consistent way soon. I don't mind if it is not uppercase, but I mind consistency. ^^

**Pull request merged**